app/controllers/api/base_controller.rb

```diff
@@ -4,9 +4,8 @@ module Api
   #
   Initializer.new.go
 
-  class BaseController < ApplicationController
```

Comment:
I agree, but if we are losing something important then it's not tested, so I should add coverage for that. I feel uncomfortable with leaving it as a black box